main.go

```diff
@@ -0,0 +1,74 @@
+package fslock
```

Comment:
I wouldnt call it main.go. Maybe fslock.go? main is semantically for binaries.

Reply:
done